packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -186,12 +176,21 @@ export default function AppHeader({
 }
 
 function MenuItem(props: {
+  name: string;
   title: string;
   onClick: () => void;
   icon: React.ReactNode;
 }) {
+  //// TODO Move ListItem to component folder && replace ListItem -> ListItemButton in MUI V5
```

Comment:
What is this about?

Reply:
ListItem should be, in my opinion, a separate component.
This is not possible with the current MUI 4, 
which is anyway deprecated.

I want to migrate to MUI 5 in my next PR